x-pack/plugins/ml/server/lib/route_guard.ts

```diff
@@ -107,20 +116,37 @@ export class RouteGuard {
         this._isMlReady
       );
 
-      return handler({
-        client,
-        request,
-        response,
-        context,
-        mlSavedObjectService,
-        mlClient: getMlClient(client, mlSavedObjectService),
-        getDataViewsService: getDataViewsServiceFactory(
-          this._getDataViews,
-          savedObjects.client,
+      const [coreStart] = await this._getStartServices();
+      // const { name, page } = coreStart.executionContext.getAsLabels();
```

Comment:
Is this code still needed?

Reply:
Removed in https://github.com/elastic/kibana/pull/148746/commits/5d139623a969d8b8092411cfa5c1932ed4b52cd5